File: CS315-HW1/Codes/altiparmak_gokberk.rb
def foo(arr)
    arr.each do |key, value|   
    puts "Key: #{key} Value: #{value}\n" end
    puts "------\n"
  end
  
  capitals = {
    "Turkey" => "Istanbul",
    "Germany" => "Berlin"
  }
  
  puts capitals['Turkey']
  foo capitals
  capitals["Italy"] = "Rome"
  foo capitals
  capitals.delete("Germany")
  foo capitals
  capitals["Turkey"] = "Ankara"
  foo capitals
  puts capitals.has_key?("Turkey")
  puts capitals.has_value?("Ankara")
  puts "------\n"
  foo capitals

File: CS315-HW1/Codes/altiparmak_gokberk.py
def foo(arr):
    for key in arr.keys():
        print("Key: " + key + " Value: " + arr[key])
    print("--------------")

def containsValue(arr, value):
    for key in arr.keys():
        if(arr[key] == value):
            return "true"
    return "false"


capitals = {"Turkey" : "Istanbul", "Germany" : "Berlin"}
foo(capitals)
capitals["Italy"] = "Rome"

foo(capitals)
print(capitals["Turkey"])

del capitals["Germany"]

foo(capitals)

capitals["Turkey"] = "Ankara"

if "Turkey" in capitals.keys():
    print("true")
else:
    print("false")

print(containsValue(capitals, "Ankara"))

foo(capitals)

